Test/CoreSDK.Test/Shared/Channel/ITelemetryTest.cs

```diff
@@ -168,7 +169,7 @@ private void SerializeWritesSequenceAsExpectedByEndpoint()
             
             TelemetryItem<TEndpointData> actual = TelemetryItemTestHelper.SerializeDeserializeTelemetryItem<TTelemetry, TEndpointData>(expected);
             
-            Assert.AreEqual(expected.Sequence, actual.Seq);
+            Assert.AreEqual(expected.Sequence, actual.seq);
```

Comment:
Is there an option we can pass to get proper looking property casing back? (or are these just fields instead of properties?)